### PR TITLE
update README to remove bad link and mark as DEPRECIATED

### PR DESCRIPTION
**Pull request by wslack**
cc @jasmeenb who reminded me to do this

### CODE REVIEW

**Comment by wslack** [comment]
@afeld to approve

File: README.md [hunk 1]
```diff
@@ -1,16 +1,9 @@
-# README
+# DEPRECIATED
```
Comment (afeld) [inline README.md]: You spelled this wrong 😉

**Review by afeld — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (afeld) [reply README.md]: Or perhaps I disagree that it lost value over time 😆

Comment (wslack) [reply README.md]: I think that's spelled right DEPRECIATED..... yeah

Comment (wslack) [reply README.md]: oh. Har.

Comment (wslack) [reply README.md]: Or should it be "RETIRED!"

Comment (afeld) [reply README.md]: These jokes are tired.

Comment (wslack) [reply README.md]: done

**Review by afeld — APPROVED** [review]

**Pull request merged**